### PR TITLE
Sim 8763 b

### PR DESCRIPTION
**Pull request by gwaldron**
Fix to ocean rendering in overhead mode.

Requires osgEarth update to 62c2fac

### CODE REVIEW

File: Examples/Ocean/Ocean.cpp [hunk 1]
```diff
@@ -155,7 +163,8 @@ static simData::ObjectId createShip(simData::DataStore& dataStore)
   {
     simData::PlatformPrefs* prefs = dataStore.mutable_platformPrefs(result, &transaction);
     prefs->mutable_commonprefs()->set_name(PLATFORM_SHIP);
-    prefs->set_icon(EXAMPLE_SHIP_ICON);
+    prefs->set_icon("H:/data/simdis/JIRA/SIM-9189/cvn68_20181023-174634/data/cvn_68.osgb");
```
Comment (emminizer) [inline Examples/Ocean/Ocean.cpp]: Looks like unintentional changes in this file

File: Examples/Ocean/Ocean.cpp [hunk 2]
```diff
@@ -155,7 +163,8 @@ static simData::ObjectId createShip(simData::DataStore& dataStore)
...
+    //prefs->set_icon(EXAMPLE_SHIP_ICON);
```
Comment (emminizer) [inline Examples/Ocean/Ocean.cpp]: Looks unintentional

File: Examples/SimpleServer/InstallOcean.cpp [hunk 3]
```diff
@@ -126,7 +128,13 @@ void InstallOcean::install(simVis::SceneManager& scene)
     triton.maxAltitude() = 30000.0f;
     triton.renderBinNumber() = simVis::BIN_OCEAN;
     osg::ref_ptr<osgEarth::Triton::TritonLayer> layer = new osgEarth::Triton::TritonLayer(triton);
+    
+    // Configure it to work in overhead mode
+    simVis::OverheadMode::configureOceanLayer(layer.get());
+
+    // Add to the map
     scene.getMap()->addLayer(layer.get());
+    //layer->getOrCreateStateSet()->setAttributeAndModes(new osg::Depth(osg::Depth::LESS, 0, 1, false), 1);
```
Comment (emminizer) [inline Examples/SimpleServer/InstallOcean.cpp]: Going to throw this into review also; can you remove dead code here?

**Comment by gwaldron** [comment]
Updated; thanks

**Pull request merged**